src/ralph/data_center/tests/factories.py

```diff
@@ -224,7 +230,13 @@ class Meta:
 
 
 class VIPFactory(DjangoModelFactory):
+    name = factory.Sequence(lambda n: 'ralph-test{}.local'.format(n))
+    # IPAddressFactory is given as string to avoid circular imports here.
+    ip = factory.SubFactory('ralph.networks.tests.factories.IPAddressFactory')
+    port = FuzzyInteger(1024, 49151)
+    protocol = factory.Iterator([VIPProtocol.TCP.id, VIPProtocol.HTTP.id])
     service_env = factory.SubFactory(ServiceEnvironmentFactory)
+    # XXX @mkurek - do we need 'parent' field here as well..?
```

Comment:
create `VIPFullFactory`, inheriting from VIPFactory, in which you'll make parent field as a SubFactory to `Cluster`